src/update.fs

```diff
@@ -0,0 +1,186 @@
+
+module UpdateCmd
+
+open System.IO
+open NUnit.Framework
+open Microsoft.Win32
+
+open PlatformHelpers
+open FSharpTestSuiteTypes
+
+type Configuration = 
+    | DEBUG
+    | RELEASE
+    override this.ToString() = 
```

Comment:
i use that for ConfigurationName argument of msbuild or some bat invocation. 
Maybe is better closer to usage instead of generic .ToString(). 
I'll move this functions closer to usage